content/ja/docs/what-is-opentelemetry.md

```diff
@@ -2,17 +2,13 @@
 title: OpenTelemetryとは
 description: OpenTelemetryが何であり、何でないかについての簡単な説明。
 weight: 150
-default_lang_commit: 93ab1301
+default_lang_commit: 44059882
 ---
 
-OpenTelemetryは[オブザーバビリティ](/docs/concepts/observability-primer/#what-is-observability)フレームワークであり、[トレース](/docs/concepts/signals/traces/)、[メトリクス](/docs/concepts/signals/metrics/)、[ログ](/docs/concepts/signals/logs/)のようなテレメトリーデータを作成・管理するためにデザインされたツールキットです。
-重要なことは、OpenTelemetryはベンダーやツールにとらわれないということです。
-つまり、[Jaeger](https://www.jaegertracing.io/)や[Prometheus](https://prometheus.io/)のようなオープンソースツールや、商用製品を含む、さまざまなオブザーバビリティバックエンドで使用できるということです。
-
-OpenTelemetryは、JaegerやPrometheusや他の商用ベンダーのようなオブザーバビリティバックエンドではありません。
-OpenTelemetryはテレメトリの生成、収集、管理、そしてエクスポートにフォーカスしています。
-OpenTelemetryの主な目的は、あなたのアプリケーションやシステムを、その言語、インフラ、ランタイム環境に関係なく、簡単に計装できるようにすることです。
-テレメトリーの保存と可視化は、意図的に他のツールに任せているということを理解するのが重要です。
+- OpenTelemetryは、[オブザーバビリティ](/docs/concepts/observability-primer/#what-is-observability)フレームワークであり、[トレース](/docs/concepts/signals/traces/)、[メトリクス](/docs/concepts/signals/metrics/)、[ログ](/docs/concepts/signals/logs/)のようなテレメトリーデータを作成・管理するためにデザインされたツールキットです。
```

Comment:
I tried applying upstream bullet expression. But I thinks the expression (`OpenTelemetry is: ... list` ) is for English so  we can keep original Japanese expressions.